README.md

```diff
@@ -19,19 +30,76 @@ pip install --upgrade pip setuptools wheel
 pip install -e .
 ```
 
+Or with a Conda virtual environment:
+
+```
+. ~/miniconda3/etc/profile.d/conda.sh
+conda create -n deepdrivemd python=3.7.9
+conda activate deepdrivemd
+pip install --upgrade pip setuptools wheel
+conda install scipy (this step is needed if a failure of installing scipy is observed)
+pip install -e .
+```
+
 Then, install pre-commit hooks: this will auto-format and auto-lint _on commit_ to enforce consistent code style:
 
 ```
 pre-commit install
 pre-commit autoupdate
 ```
 
-In some places, DeepDriveMD relies on external libraries to configure MD simulations and import specific ML models.
+2. Install OpenMM:
+
+- by source code (for Linux ppc64le, e.g., Summit)
+https://gist.github.com/lee212/4bbfe520c8003fbb91929731b8ea8a1e
+
+- by conda (for Linux x86\_64, e.g., PSC Bridges)
+```
+module load anaconda3
+module load cuda/9.2
+source /opt/packages/anaconda/anaconda3-5.2.0/etc/profile.d/conda.sh
+conda install -c omnia/label/cuda92 openmm
+```
+
+3. In some places, DeepDriveMD relies on external libraries to configure MD simulations and import specific ML models.
 
 For MD, install the `mdtools` package found here: https://github.com/braceal/MD-tools
 
+```
+git clone https://github.com/braceal/MD-tools.git
+pip install .
+```
+
 For ML (specifically the AAE model), install the `molecules` package found here: https://github.com/braceal/molecules/tree/main
 
+```
+git clone https://github.com/braceal/molecules.git
+pip install .
+```
+
+#### Stage: machine_learning
+
+1. Install the `deepdrivemd` virtual environment as above (`deepdrivemd` is needed in all the virtual environments since each task uses the DDMD_API to communicate with the outputs of other tasks).
+
+2. Install the `keras-CVAE` model with `rapidsai DBSCAN` package found here: https://www.ibm.com/docs/en/wmlce/1.6.2?topic=installing-mldl-frameworks
+
+```
+conda config --prepend channels https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/
+conda install powerai-rapids
```

Comment:
Isn't this for IBM PowerPC? other platforms may use `conda install rapids` to get cuML.

Reply:
Yes I believe so, but I have not tested it on other platforms. Perhaps we can add documentation for other platforms as we increase our support of those platforms. It might be a good idea to have an installation page in the docs for each platform. Or if not a full page, at least a section within the main installation page.